test/adapters/file/file.spec.js

```diff
@@ -340,19 +340,19 @@ describe('file adapter tests', function () {
         });
 
 
-        it('fails when you attempt to write to a file with more points than than -bufferLimit allows', function() {
+        it('fails when you attempt to write to a file with more points than than -limit allows', function() {
             return check_juttle({
                 program: 'emit -limit 3 -from :2014-01-01: | write file -file "' + filename + '"'
             })
             .then(function() {
                 return check_juttle({
-                    program: 'emit -limit 1 -from :2014-01-01: | write file -file "' + filename + '" -bufferLimit 2'
+                    program: 'emit -limit 1 -from :2014-01-01: | write file -file "' + filename + '" -limit 2'
                 });
             })
             .then(function(result) {
                 expect(result.errors.length).to.equal(1);
                 expect(result.warnings.length).to.equal(0);
-                expect(result.errors[0]).to.contain('Error: option bufferLimit exceeded limit of 2, droppping points.');
+                expect(result.errors[0]).to.contain('Error: option limit exceeded limit of 2, droppping points.');
```

Comment:
Not part of this change, but I noticed there's a typo in `droppping`. Might as well fix that along the way.